packages/flutter_tools/templates/create/ios.tmpl/Flutter/Debug.xcconfig

```diff
@@ -0,0 +1,2 @@
+#include "Generated.xcconfig"
+#include "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"
```

Comment:
Should these Debug.xcconfig and Release.xcconfig files be in the Flutter directory?  Where are they in a normal cocoapods project?